src/neo/Ledger/Blockchain.cs

```diff
@@ -63,7 +63,7 @@ public class RelayResult { public IInventory Inventory; public VerifyResult Resu
         private readonly Dictionary<UInt256, Block> block_cache = new Dictionary<UInt256, Block>();
         private readonly Dictionary<uint, LinkedList<Block>> block_cache_unverified = new Dictionary<uint, LinkedList<Block>>();
         internal readonly RelayCache ConsensusRelayCache = new RelayCache(100);
-        private SnapshotView currentSnapshot;
+        public SnapshotView currentSnapshot;
```

Comment:
I am not sure about making this `public`.

Reply:
Maybe instead we can add a public get method?

Reply:
Hi Vitor, could you have a look at corresponding change? Thx:)